lib/rust/metamodel/src/java/from_meta.rs

```diff
@@ -106,7 +106,9 @@ impl FromMeta {
         let mut fields = Vec::with_capacity(fields_.size_hint().0);
         for field in fields_ {
             let meta::Field { name, type_, hide, .. } = field;
-            let name = name.to_camel_case().expect("Unimplemented: Tuples.");
+            let mut name_ = meta::FieldName::from_snake_case("field");
```

Comment:
This change renamed even the public getters. Now I see `Line.getFieldExpression()` methods & co.